Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -236,28 +239,23 @@ extension NavigationMapView {
 
                 let lineString = feature.geometry.value as? LineString
                 guard let distance = lineString?.distance() else { return gradientStops }
+                let stopGap = max(min(GradientCongestionBlendingDistance, distance * 0.1) / routeDistance, 0.0000000000000002)
                 
```

Comment:
Is there a risk of division by zero if the linestring somehow ends up being zero-length?

Reply:
Yes, your concern is correct, I'll add check for it.